Application has thrown an uncaught exception and is terminated:
C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\controllers\qrapp.js:81
		};
		 ^
SyntaxError: Unexpected token ;
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\node_modules\express-load\lib\express-load.js:235:15
    at iterate (C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\node_modules\async\lib\async.js:131:13)
    at C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\node_modules\async\lib\async.js:142:25
    at C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\node_modules\express-load\lib\express-load.js:263:7
    at iterate (C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\node_modules\async\lib\async.js:131:13)
    at Object.async.eachSeries (C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\node_modules\async\lib\async.js:147:9)
    at ExpressLoad.into (C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\node_modules\express-load\lib\express-load.js:230:9)
    at Object.<anonymous> (C:\inetpub\wwwroot\App1_App2\www\servicesctrl_dev\index.js:29:3)
